python/lsst/sconsUtils/tests.py

```diff
@@ -293,3 +305,38 @@ def junitPrefix(self):
             prefix += ".{0}".format(os.environ[controlVar])
 
         return prefix
+
+    def _pytestCoverage(self, prefix="pytest"):
+        """Form the additional arguments required to enable coverage testing.
+        Prefix is used to form the output files.
+        """
+
+        options = ""
+
+        # if we have a standard package layout, restrict coverage to the
+        # python code and do not include the tests.
+        if os.path.exists("python") or os.path.exists(os.path.join(os.path.pardir, "python")):
+            options += " --cov=python"
+        else:
+            # Default to coverage for everything in this directory and below
+            options += " --cov=."
```

Comment:
What packages do we have that would end up in this branch?

Reply:
I can't remember why I needed that. I'm using `--cov=.` for everything now.